Vertexvis/work-instructions-demo
# PLAT-4646 Replace experimental ghosting

## Summary
This update:
- replaces experimental ghosting with phantom rendering
- upgrades to Vertex SDK 0.22.0 and Vertex Node Client 0.23.0
- upgrades to Node 20, React 18, and NextJS 14 

## Test Plan
1. Start the docker container and launch the demo in a browser window.
2. Navigate through the instructions by clicking the steps. Ensure views change as expected.
3. Right-click on a part in the view to render as phantom
4. Reset the scene by clicking the Reset Scene button in the viewer toolbar

## Release Notes

Replaces experimental ghosting with phantom rendering
- Right-click on a part to render as phantom

Upgrades several dependencies, including:
- Vertex Viewer to 0.22.0
- Vertex Node client to 0.23.0
- React to 18.x
- NextJs to 14.x
- Node to 20.x

Adds a reset scene button to the viewer toolbar

## Possible Regressions
None identified

## Dependencies
PLAT-4646
